db/migrate/20231201204221_add_is_published_to_pets.rb

```diff
@@ -0,0 +1,5 @@
+class AddIsPublishedToPets < ActiveRecord::Migration[7.0]
```

Comment:
I appreciate seeing the history on the migrations, but could you condense this into one migration?
I agree with naming the column `published` with default of `false` and null false.

you should be able to delete all three, make a new one, drop your local db and rerun migrations.